server/Gemfile

```diff
@@ -17,6 +17,8 @@ gem "fast_jsonapi"
 gem "oj", "~> 3.3", ">= 3.3.5"
 gem "pg", "1.3.0.rc3"
 gem "rack-cors"
+gem "cloudinary"
+gem 'activestorage-validator'
```

Comment:
adding new gems normally require to add Gemfile.lock, please run `bundle install` and add Gemfile.lock with appropriate changes